packages/main/src/mainWindow.ts

```diff
@@ -105,6 +106,10 @@ async function createWindow() {
     browserWindow.destroy();
   });
 
+  contextMenu({
+    showInspectElement: import.meta.env.DEV,
+  });
+
```

Comment:
For developer mode I think it make sense for the inspect even if the developer console is always there but then for production it brings other menus that probably should not be displayed
![image](https://user-images.githubusercontent.com/436777/173001981-ca092837-5ca2-4d9b-bb45-dace34ce7de2.png)



```suggestion
  if (import.meta.env.DEV) {
  contextMenu({
    showInspectElement: true,
  });
}
```

Reply:
There are options you do want to keep for production too, let's decide, these you do want

```
cut
copy
paste
```

https://github.com/sindresorhus/electron-context-menu#readme